workshop/permissions.py

```diff
@@ -76,3 +77,33 @@ def has_object_permission(self, request, view, obj):
                 or obj.workshop in profile.organized_workshops.all()):
             return True
         return False
+
+class AllowParticularClubHead(permissions.BasePermission):
+    message = "You are not authorized to perform this task"
+
+    def has_object_permission(self, request, view, obj):
+        if request.method in permissions.SAFE_METHODS:
+            return True
+        if not request.user.is_authenticated:
+            return False
+        # pylint: disable=no-member
+        profile = UserProfile.objects.get(user=request.user)
+        club = obj
+        if club in profile.get_club_privileges:
+            return True
+        return False
+
+class AllowParticularCouncilHead(permissions.BasePermission):
+    message = "You are not authorized to perform this task"
+
+    def has_object_permission(self, request, view, obj):
+        if request.method in permissions.SAFE_METHODS:
+            return True
+        if not request.user.is_authenticated:
+            return False
+        # pylint: disable=no-member
+        profile = UserProfile.objects.get(user=request.user)
+        council = obj
+        if council in profile.get_council_privileges:
+            return True
+        return False
```

Comment:
```suggestion
        return False

```
Please merge this branch with the latest master, so that lint checks can run, and fix the lint check errors (if any).